### PR TITLE
proxmark3: version bump & fixup

### PR DESCRIPTION
**Pull request by msoose**
Added dependency
Update to latest release version
Follow standards better

### CODE REVIEW

**Review by thunder-coding — CHANGES_REQUESTED** [review]

**Comment by buttaface** [comment]
Sorry, didn't see this and just pushed a version bump. If you will adapt this patch to that one and make this a revision bump instead, I will review and get this in.

File: packages/proxmark3/build.sh [hunk 1]
```diff
@@ -5,6 +5,14 @@ TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
...
 TERMUX_PKG_SRCURL=https://github.com/RfidResearchGroup/proxmark3/archive/v${TERMUX_PKG_VERSION:2}.tar.gz
 TERMUX_PKG_SHA256=49c4f1854b364aa7ea7083581351f867128e71ea783d0ecd71fc41bcf7f63584
+TERMUX_PKG_DEPENDS="libbz2, libc++, readline"
+TERMUX_PKG_BUILD_IN_SRC="true"
+TERMUX_PKG_BLACKLISTED_ARCHES="i686, x86_64"
+
+termux_step_post_configure() {
+        LDLIBS="-L${TERMUX_PREFIX}/lib" INCLUDES_CLIENT="-I${TERMUX_PREFIX}/include" CFLAGS="-I${TERMUX_PREFIX}/include"
+        TERMUX_PKG_EXTRA_MAKE_ARGS="client -j $TERMUX_MAKE_PROCESSES CC=$CC CXX=$CXX LD=$CXX cpu_arch=generic SKIPREVENGTEST=1 SKIPQT=1 SKIPBT=1 SKIPPTHREAD=1 SKIPGPU=1 PLATFORM=PM3GENERIC"
+
```
Comment (suhan-paradkar) [inline packages/proxmark3/build.sh:15]: forgot an end bracket here...
also... why is this repeated again in termux_step_make ??

File: packages/proxmark3/build.sh [hunk 2]
```diff
@@ -5,6 +5,14 @@ TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
 TERMUX_PKG_VERSION="1:4.13441"
 TERMUX_PKG_SRCURL=https://github.com/RfidResearchGroup/proxmark3/archive/v${TERMUX_PKG_VERSION:2}.tar.gz
 TERMUX_PKG_SHA256=49c4f1854b364aa7ea7083581351f867128e71ea783d0ecd71fc41bcf7f63584
+TERMUX_PKG_DEPENDS="libbz2, libc++, readline"
+TERMUX_PKG_BUILD_IN_SRC="true"
+TERMUX_PKG_BLACKLISTED_ARCHES="i686, x86_64"
```
Comment (suhan-paradkar) [inline packages/proxmark3/build.sh:10]: why is this repeated again in lines 16 17 and 18??

**Review by suhan-paradkar — CHANGES_REQUESTED** [review]

**Comment by buttaface** [comment]
Hold off a bit: since my commit failed to build, I cleaned up the changes here and pushed that, dca725d, let's see if that works.

File: packages/proxmark3/build.sh (hunk 2 above)
Comment (msoose) [reply packages/proxmark3/build.sh:10]: Because apparently I can't copy and paste properly

**Comment by buttaface** [comment]
Sigh, my attempt to reorganize it didn't work, going back to how you had it, ce32b8c.

**Comment by buttaface** [comment]
Sigh, I thought you tested this change, I will build it locally and fix it.

**Comment by msoose** [comment]
This was tested, but I completely botched a commit. I'm going to just close this.

**Comment by buttaface** [comment]
Oh, my mistake, maybe it worked in your tests because you set the `CFLAGS` variable, whereas I didn't need that when I built this locally. I will narrow it down and push the fix.

**Comment by buttaface** [comment]
Fixed in eed9d41.

**Pull request closed without merging**